ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -24,7 +25,8 @@
 '''
 
 LOOKAHEAD_WPS = 50  # Number of waypoints we will publish. You can change this number
-FREQ_DELAY = .001
+FREQ_DELAY = .01
```

Comment:
Do you think this lookahead is needed anymore?

Reply:
I don't think. I forgot to delete this.